src/webui/webapplication.cpp

```diff
@@ -943,6 +947,104 @@ void WebApplication::removeCategories()
         BitTorrent::Session::instance()->removeCategory(category);
 }
 
+void WebApplication::addRSSFolder()
+{
+    CHECK_PARAMETERS("path");
+
+    const QString path = request().posts["path"].trimmed();
+    QString error;
+    if (!RSS::Session::instance()->addFolder(path, &error)) {
+        status(409, "Conflict");
+        print(error, Http::CONTENT_TYPE_TXT);
+    }
+}
+
+void WebApplication::addRSSFeed()
+{
+    CHECK_PARAMETERS("url" << "path");
+
+    const QString url = request().posts["url"].trimmed();
+    const QString path = request().posts["path"].trimmed();
+    QString error;
+    if (!RSS::Session::instance()->addFeed(url, (path.isEmpty() ? url : path), &error)) {
+        status(409, "Conflict");
+        print(error, Http::CONTENT_TYPE_TXT);
+    }
+}
+
+void WebApplication::removeRSSItem()
+{
+    CHECK_PARAMETERS("path");
+
+    const QString path = request().posts["path"].trimmed();
+    QString error;
+    if (!RSS::Session::instance()->removeItem(path, &error)) {
+        status(409, "Conflict");
+        print(error, Http::CONTENT_TYPE_TXT);
+    }
+}
+
+void WebApplication::moveRSSItem()
+{
+    CHECK_PARAMETERS("itemPath" << "destPath");
+
+    const QString itemPath = request().posts["itemPath"].trimmed();
+    const QString destPath = request().posts["destPath"].trimmed();
+    QString error;
+    if (!RSS::Session::instance()->moveItem(itemPath, destPath, &error)) {
+        status(409, "Conflict");
+        print(error, Http::CONTENT_TYPE_TXT);
+    }
+}
+
+void WebApplication::getRSSItems()
+{
+    print(QJsonDocument(
+              RSS::Session::instance()->rootFolder()->toJsonValue(
+                  parseBool(request().gets["withData"], false)
```

Comment:
please use a named variable for this line, to ease documentation work (easier to track).